### ISSUE TITLE
Quotation by Email

### ISSUE THREAD

**Issue by cosimot** [frappe/erpnext#13293]
Sending quotation by email we often are blocked by anti spam filters, the result is that we have to follow-up or the client has to solicit.
What if, when we select "Send Read Receipt" the receipt is sent to "Reply To" email address?
Like this the quotation status could change from "Submitted" to "Read".
Could it be done eventually?
@PawanMeh  what do you think?

**Comment by joolsr** [comment]
There are many factors that can affect whether email are blacked by antispam filters, you can test for this with a number of third party tools freely on the internet. It depends on how you installed the application as to how the emails are sent.

**Comment by cosimot** [comment]
yes but it also depends on how users set their filters, when there is an attachment and a hyperlink cases are very high... that is why people also in business are moving to WhatsApp or Facebook Messenger.

**Comment by DXBD** [comment]
> yes but it also depends on how users set their filters, when there is an attachment and a hyperlink cases are very high... that is why people also in business are moving to WhatsApp or Facebook Messenger.

I would argue that anyone who is serious about business will be avoiding ANYTHING to do with facebook. Especially given the recent news about how they openly give data away to "partners"

I do agree however that social media is going to change back to chat / group based solutions.
